pdr_backend/accuracy/app.py

```diff
@@ -239,12 +248,31 @@ def calculate_timeframe_timestamps(
         # else timedelta(days=1)
     )
     start_ts = UnixTimeS(int((datetime.utcnow() - time_delta).timestamp()))
-
     return start_ts, end_ts
 
 
 @enforce_types
-def save_statistics_to_file():
+def fetch_statistics_using_ETL():
```

Comment:
shouldn't the ppss be customisable?

Reply:
I think it's good as it is for now, the scope of the API is quite specific